edx/xblock-lti-consumer
# [BD-2] BB-2365: Merge LTI 1.3 support to master.

This PR merges the feature branch that implements LTI 1.3 launch support.
Already tested and approved by edX team on review branch.

**Testing instructions:**
1. Clone this repo and install it in your devstack.
2. Follow the setup instructions on https://github.com/edx/xblock-lti-consumer/blob/d020659fe602c251d55f8ad801e14376790b6571/README.rst to enable and setup a LTI 1.3 integration.
3. Check that a LTI 1.3 Launch is performed correctly:
![success](https://user-images.githubusercontent.com/27893385/85781285-e730c100-b6fb-11ea-8908-6a63b9ccd26e.png)


**Reviewers:**
- [ ] @__xitij2000__
